docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,349 @@ have not been included. These are:
+    - Incorrect select 
+    - 
+  * - Sex 
+    - Census, Household Surveys, WIC 
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Race/Ethnicity 
+    - Census, WIC
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - SSN
+    - Taxes (both), SSA
+    - 1%
+    - 1
+    - "Borrowed" SSN, Copy from within Household, Numeric miswriting 
+    - Note that not all types of noise apply to all observers, details below 
+  * - ITIN
+    - Taxes 1040
+    - 1%
+    - 1
+    - Copy from within Household, Numeric miswriting 
+    - Note that not all types of noise apply to all observers 
+  * - Income / Wages
+    - Taxes (both)
+    - 1%
+    - 1
+    - Numeric miswriting
+    - Note that wages and income are on separate tax forms and noise is applied to each separately 
+  * - Employer ID 
+    - Taxes (both)
+    - 1%
+    - 1
+    - Numeric miswriting
+    - 
+  * - Employer Name 
+    - Taxes (both)
+    - 1%
+    - 1
+    - OCR, typographic 
+    - 
+  * - Type of Tax Form  
+    - Taxes (both)
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Type of SSA Event 
+    - SSA 
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Date of SSA Event 
+    - SSA 
+    - 1%
+    - N/A
+    - Numeric miswriting, month and day swap 
+    - 
+
+The below section further describes types of noise including any code 
+available and information for implementation. **Software engineering team - please alert the research team if any of the below looks to be particularly challenging for further discussion.**
+
+**Converting from Noise Inputs to Noise Function Parameters for OCR, Phonetic, Typographic and Numeric Miswriting** 
+
+For simplicity for the end user, they will input two values for noise level: 
+
+#. Rows with Errors 
+#. Errors per Row 
+
+For types of noise others than those listed above, the only parameter is rows with error 
+and can be followed exactly. For noise types above, the functions we will use take different parameters: 
+
+#. Rows Eligible for Errors 
+#. Error Rate per Character OR per Token 
+
+These parameters, while related are different in key ways and therefore we need to 
+provide a way to convert between them. 
+
+Starting with #1 from both lists, we will calculate the number of rows eligible 
+for errors to approximate the user inputted rows with errors given. To calculate 
+this we can: 
+
+#. Compute the average length of strings in the column 
+#. Probability of at least 1 error = 1 - ((1 - (user defined errors per row / average string length))^(average string length))
+#. Rows eligible for errors = User defined rows with errors / probability of at least 1 error 
+
+Next, once a row is selected to receive noise, we define how much noise it should receive. 
+The default value here is 1 error per row, but can be changed. To calculate the error rate 
+per character/token, we can use this equation: 
+
+Error rate per Character/Token = (user defined errors per row)/(average string length)
```

Comment:
@SylLutze @zmbc @aflaxman This was just an idea I wrote down on the fly, and I'm not entirely sure it is the best strategy, for a few different reasons.

I was going to send a message in Slack this morning, but then I was out because of sick kids: I suggest that the researchers meet to discuss how we want to parameterize the noise functions for the end user in pseudopeople, and whether the version we specify for the minimal model should be the same as what we want in the final product or whether we want to do something simpler for now. (I think how we choose to simplify things for the MVM might depend on what we eventually want to do, but I'm not clear on what the best strategy is for the final product from a usability perspective.)

Reply:
My original thought was that the simplest thing to do for for the MVM would be essentially what Syl wrote down in the first place: Allow the user to specify the fraction of eligible rows, and for now have the researchers choose fixed values for the per-token error probabilities (which was left as a to-do because I hadn't thought about what good default values would be). The main problem with the original writeup was that we need to be clear for the engineers that it's token-level probabilities instead of character-level, and we should be clear for the engineers and the end user that the user is specifying the probability of *eligibility* for errors rather than the probability of actually getting an error, and we could give some estimates of what the difference would be under certain assumptions.

For the final product, I think we should do something more similar to what is proposed here, but I think we can do a better job about having parameters that are useful and intuitive to the user, and having code to accurately match these instead of just saying that everything is approximate and that's a limitation. But there are several details to work out.

Reply:
@NathanielBlairStahn First of all, I agree with you that this doesn't need to be worked out right now. I think we are confident that this will work "under the hood" using a probability of eligibility, and then a per-character or per-token (depending on noise type) error rate among the eligible. I **do** think that both (not just the former) should be customizable by the user.

All the additional stuff is for usability: allowing the user to set parameters with more intuitive meaning than those two. What I proposed yesterday to Syl was to have the user set these two parameters:

- Percent of rows with any error
- Mean number of errors per incorrect row

I think these have intuitive significance in a way that "eligibility" does not.

The math of how to get from those to probability of eligibility and per-token error rate is not simple and I believe is not correct in this PR. I _think_ I've mostly worked it out on paper (so not in a form yet that I can share). But this will take more work and more in-depth review.

@SylLutze For this PR I think we should just explain the version with eligibility probability and per-token error rate and add a TODO that we want to allow the user to specify more intuitive parameters from which we would back-calculate the easier-to-implement ones.